src/commands/init.js

```diff
@@ -202,6 +213,71 @@ const addToPackageScripts = async (cwd: string) => {
   );
 };
 
+const addToGradleBuild = async (cwd: string) => {
+  let entry;
+
+  // Does `android/app/build.gradle` exist?
+  const androidPath = path.join(cwd, 'android/app');
+  if (fs.existsSync(androidPath)) {
+    const list = fs
+      .readdirSync(androidPath)
+      .filter(file => file.includes('build.gradle'));
+
+    // Do nothing if multiple projects were found
+    if (list.length === 1) {
+      entry = path.join(androidPath, list[0]);
+    }
+  }
+
+  // Otherwise, ask for path to a file
+  if (!entry) {
+    const result = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'entry',
+        message: 'Enter path to the android/app/build.gradle file',
+        validate: pathToFile =>
+          fs.existsSync(pathToFile) && pathToFile.includes('build.gradle')
+            ? true
+            : `${pathToFile} is not a valid build.gradle`,
+      },
+    ]);
+
+    entry = path.resolve(cwd, result.entry);
```

Comment:
No need for `cwd` here, `resolve` will automatically use `process.cwd()` if the path is not absolute, so either use `path.resolve(result.entry)` or `path.join(cwd, results.entry)`

Reply:
Just borrowed from the existing addToXcodeBuild. Do you want me to refactor that instance as well?

Reply:
It's more of a nit picking, but if you're fine with it, refactoring it would be nice

Reply:
Just trying to follow the existing conventions :)